### PR TITLE
show notes for 'How Computers Work' series

### PR DESCRIPTION
**Pull request by tanyaparker**

### CODE REVIEW

File: dashboard/config/locales/slides.en.yml [hunk 1]
```diff
@@ -2020,3 +2020,204 @@ en:
...
+      2:
+        image: 'notes/csd_concept_how_computers_work_2.png'
+        text: "After that, you'll learn how computers use circuits to do everything from simple math to simulating entire virtual worlds. Next, you'll zoom out and look at the different parts of a computer that actually input, output, and store all that information. Finally, you'll take a look at what code actually is and how software controls hardware. So whether you're just curious about the devices you use every day or you want to design the innovations of the future, the first step is learning how computers work."  
+    csd_concept_computer:
+      1:
+        image: 'notes/csd_concept_computer_1.png'
+        text: "My name is May-Li Khoe and I'm a designer and an inventor. So some of the things I've designed have been at Apple, and now I design products for kids to use so that they can have an easier time in school. My other jobs include DJ-ing and dancing."
```
Comment (caleybrock) [inline dashboard/config/locales/slides.en.yml:2033]: is this correct name spelling?

File: dashboard/config/locales/slides.en.yml [hunk 2]
```diff
@@ -2020,3 +2020,204 @@ en:
...
+        text: "All of these images are made out of teeny dots called pixels, and each pixel has a color. Each of the colors can be represented with numbers. When you consider the typical image has millions of these pixels, and the typical video shows 30 images per second now we're talking about a lot of data here."
+      12:
+        image: 'notes/csd_concept_binary_data_12.png'
+        text: "Every sound is basically a series of vibrations in the air. Vibrations can be represented graphically as a waveform. Any point on this waveform can be represented by a number. And this way, any sound can be broken down into a series of numbers. If you want higher quality sound, you will pick 32-bit audio or an 8-bit audio. More bits means a higher range of numbers."
+      13:
+        image: 'notes/csd_concept_binary_data_13.png'
+        text: "When you use a computer to write code or make your own app, you're not dealing directly with these ones and zeros. But you will be dealing with images or sound or video. So if you want to understand how computers work on the inside, it all comes down these simple ones and zeros and the electrical signals in the circuits behind them "
```
Comment (caleybrock) [inline dashboard/config/locales/slides.en.yml]: nit: missing period

File: dashboard/config/locales/slides.en.yml [hunk 3]
```diff
@@ -2020,3 +2020,204 @@ en:
...
+      15:
+        image: 'notes/csd_concept_hardware_software_15.png'
+        text: "Inside every computer is an operating system managing software that controls the computer's hardware."
+      16:
+        image: 'notes/csd_concept_hardware_software_16.png'
+        text: "The software is a series of commands made of simple binary code, and that binary code is just electrical signals flowing through billions of tiny circuits."
+      17:
+        image: 'notes/csd_concept_hardware_software_17.png'
+        text: "Computers have the potential to do all kinds of amazing things. But the only thing that makes the computer smart, or useful, is you. When you learn to code you get to define the problem you want to solve, and write the software that turns those ideas into reality. "
```
Comment (caleybrock) [inline dashboard/config/locales/slides.en.yml]: super nit: extra space at the end of this line

File: dashboard/config/locales/slides.en.yml (hunk 1 above)
Comment (tanyaparker) [reply dashboard/config/locales/slides.en.yml:2033]: Yup! [Linked In](https://www.linkedin.com/in/maylikhoe/)

**Pull request merged**